connector-sdk/test/src/main/java/io/camunda/connector/test/TestDocumentMetadata.java

```diff
@@ -0,0 +1,78 @@
+package io.camunda.connector.test;
```

Comment:
This file is basically a copy of DocumentMetadataImpl.

Let me which option you like more:
:one: Copy DocumentMetadataImpl to TestDocumentMetadata
:two: Move DocumentMetadataImpl from runtime-core to sdk/core

Reply:
@chillleader If its in the SDK I would prefer an interface as it probably should be constructed by client code.

Reply:
It's not supposed to be constructed by client code, the client passes all data in `DocumentCreationRequest`, and this class is just part of the document reference model

Reply:
So  I would then remove the DocumentMetadata interface and create a class DocumentMetadata in the runtime, right?
But then how would I handle `DocumentReferenceModel` in sdk/jackson-datatype-document using the current DocumentMetadata interface?

Reply:
Yes, I would prefer interfaces in the SDK as much as possible.

Yes, basically 1.